app/main.py

```diff
@@ -1,2 +1,52 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 8
+        self.load_factor = 2 / 3
+        self.dictionary = [-1 for _ in range(8)]
```

Comment:
```suggestion
        self.dictionary = [None for _ in range(8)]
```
better to use `None` in this case